content-src/components/Base/Base.jsx

```diff
@@ -186,7 +186,7 @@ export class BaseContent extends React.PureComponent {
                 }
               {isDiscoveryStream ? (
                 <ErrorBoundary className="borderless-error">
-                  <DiscoveryStreamBase />
+                  <DiscoveryStreamBase isDarkModeMessage={prefs.darkModeMessage} />
```

Comment:
@k88hudson will this work or is there a better way to get prefs.darkModeMessage inside DiscoveryStreamBase.jsx to avoid passing it as props?